### ISSUE TITLE
Start screen for initialization of settings

### ISSUE THREAD

**Issue by Harkishen-Singh** [Harkishen-Singh/Jarvis-personal-assistant#8]
This would ask the user for all the required details for the bot to work, on the first use of the application.
Includes : 
1. Welcome screen
2. Name
3. Use purpose
4. root password
5. Github/Gitlab password

any credentials left out, please mention as a comment. Discussion

**Comment by gunjanmimo** [comment]
Sir, can I solve this issue with HTML CSS js?

**Comment by muskankhedia** [comment]
Yes, you can, it will also require to AngularJs in some part to store the data. Till then you can go for UI designing.
